chef_master/source/troubleshooting_chef_automate.rst

```diff
@@ -62,12 +62,29 @@ If you already have and are still seeing that error, then it's time to see if th
 registered with the chef server correctly.  What correctly means in this case is something
 that matches your build node query.
 
-By default, Chef Automate build nodes/runners generated by ``automate-ctl install-build-node`` or ``automate-ctl install-runner``, respectively are tagged
-as ``delivery-build-node``. If your delivery.rb contains a custom search query
-(``delivery['default_search']`` is set), try appending ``" OR tags:delivery-build-node"`` to your query.
+By default, Chef Automate build nodes/runners generated by ``automate-ctl install-build-node`` or ``automate-ctl install-runner``, respectively are tagged as ``delivery-build-node`` and ``delivery-job-runner``. If your delivery.rb contains a custom search query (``delivery['default_search']`` is set), try appending ``" OR tags:delivery-build-node"`` or ``" OR tags:delivery-job-runner"`` to your query.
 
-If you are trying to debug a particular build node and want to ensure that one is available for your projects,
-then modify the build-nodes default search for your project as described in :doc:`Configure a Project </config_json_delivery>`.
+The short story about build-node and runner configuration is this at minimum.
+
+If your Automate system has Builders(Push Jobs), then your projects should have this config item in .delivery/config.json
+
+   .. code-block:: none
+   
+       "job_dispatch": {
+         "version": "v1"
+       },
+  
+If your Automate system has Runners, then your projects should have this config item in .delivery/config.json
+
+   .. code-block:: none
+   
+       "job_dispatch": {
+         "version": "v2"
+       },
+
+
+For further detail: If you are trying to debug a particular build node or runner and want to ensure that one is available for your projects,
```

Comment:
Hi,  
I need clarification on "want to ensure that one is available for your projects".  Which do you mean:
1) "want to ensure that an alternative build node or runner is available for your projects"
2) "want to ensure that the build node or runner that you are trying to debug is available for your projects"

The ''build nodes'' option could be used either way--but which did you intend?

Thanks!